app/views/search/_suggest_leader.html.erb

```diff
@@ -0,0 +1,16 @@
+<% users.each do |user| %>
+  <li class="uk-panel-box uk-grid uk-grid-collapse dark user-search user-result hover uk-padding-remove" onclick="window.location='<%= profile_path(user.screen_name) %>'">
```

Comment:
```suggestion
  <li class="uk-panel-box uk-grid uk-grid-collapse dark user-search user-result hover uk-padding-remove">
```
The `onclick` handler is unnecessary here, it also makes funny things if you click on the list item instead of the button.